packages/shared/src/components/cards/squad/SquadsDirectoryFeed.tsx

```diff
@@ -68,6 +70,22 @@ export function SquadsDirectoryFeed({
           />
         ),
       )}
+      {isInitialLoading && !flatSources.length && (
+        <>
+          <PlaceholderSquadGrid
+            className="max-w-80"
+            isFeatured={query.featured}
+          />
+          <PlaceholderSquadGrid
+            className="max-w-80"
+            isFeatured={query.featured}
+          />
+          <PlaceholderSquadGrid
+            className="max-w-80"
+            isFeatured={query.featured}
+          />
```

Comment:
The placeholder should not be based only on query.featured on squad directory feed, related to this issue https://dailydotdev.slack.com/archives/C069XNYMDGV/p1726757281431199

Reply:
This is not the single `featured` coming from the squad's object. 
This comes from the query settings, so if `query.featured` is true there will be only featured cards 👀

Reply:
Ah cool then just making sure